README.md

```diff
@@ -25,21 +18,20 @@ npm install ra-supabase
 // in supabase.js
 import { createClient } from '@supabase/supabase-js';
 
-export const supabase = createClient(SUPABASE_URL, SUPABASE_ANON_KEY);
+export const supabase = createClient('YOUR_SUPABASE_URL', 'YOUR_SUPABASE_ANON_KEY');
 
 // in dataProvider.js
-import { supabaseDataProvider } from 'ra-supabase';
-import { supabase } from './supabase';
+import { supabaseDataProvider } from 'ra-supabase-core';
```

Comment:
This isn't what's inside the demo `('ra-supabase' instead of `'ra-supabase-core').``
=> Is this snippet wrong or is the demo wrong?

Reply:
Besides, since `ra-supabase` seems to re-export everything, why did you need this change?